domain-model/src/test/java/uk/gov/hmcts/cmc/domain/amount/TotalAmountCalculatorTest.java

```diff
@@ -99,7 +99,7 @@ public void totalTillDateOfIssueShouldReturnClaimAmountPlusFeeWhenNoInterest() {
     @Test
     public void totalTillTodayShouldReturnCalculatedTotalAmountPlusInterestWhenCalculateFromSubmission() {
```

Comment:
Please update description to highlight that interest is calculated from issue date, not submission date.

Reply:
done